### DISCUSSION TITLE
Updating JS Dependencies

### DISCUSSION THREAD

**Discussion by DerMistkaefer** [pelican-dev/panel#509] [General]
My great hope with Pelican was that the JS dependencies would finally be updated. Unfortunately, I have found that there has been no improvement in this area, similar to Pterodactyl.

I am also willing to contribute to this, but I wanted to clarify if there are any specific reasons why this has not been addressed yet.

First, we need to update Webpack, as it currently prevents us from updating to newer JavaScript packages. Webpack 5 has been available since 2020.

Alternatively, Pterodactyl v2 used Vite. However, I'm unsure if we should switch to yet another new technology.

**Comment by DerMistkaefer** [comment]
Update: I just noticed that you are already working on Vite (https://github.com/pelican-dev/panel/pull/493).

**Comment by rmartinoscar** [comment]
Hey it's on the roadmap no ETA
